docs/src/content/docs/getting-started/installation.mdx

```diff
@@ -5,29 +5,33 @@ description: Learn how to install astro-integration-kit in your project.
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
+:::caution
+Astro Integration Kit is unstable, breaking changes can occur at anytime.
```

Comment:
```suggestion
Astro Integration Kit iterates very quickly as we are constantly improving it's APIs. Breaking changes can occur at any time.
```

Like don't get me wrong, it _is_ unstable 😂 But maybe we should soften this a _little_ bit!

Reply:
xd yeah you're right!